packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -146,35 +180,97 @@ export class GCFBootstrapper {
     return config as Options;
   }
 
+  parseRequestHeaders(
+    request: express.Request
+  ): {name: string; id: string; signature: string; taskId: string} {
+    const name =
+      request.get('x-github-event') || request.get('X-GitHub-Event') || '';
+    const id =
+      request.get('x-github-delivery') ||
+      request.get('X-GitHub-Delivery') ||
+      '';
+    // TODO: add test to validate this signature is used on the initial
+    // webhook from GitHub:
+    const signature =
+      request.get('x-github-delivery') ||
+      request.get('X-GitHub-Delivery') ||
+      '';
```

Comment:
Strange that these are both reading the same value, although it was in the previous version too.

Reply:
Yeah I'm not sure if it was intentional or not, but I left it as-is